torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1945,10 +1945,10 @@ def hint():
                     msg = f"  {len(error_msgs) + 1}. {msg()}"
                     error_msgs.append(msg)
             if len(error_msgs) > 0:
-                log.warning("Warning only constraints violated %s", warn_msgs)
+                log.info("Warning only constraints violated %s", warn_msgs)
                 raise ConstraintViolationError(f"Constraints violated!\n{error_msgs}")
             elif len(warn_msgs) > 0:
-                log.warning("%s Warning only constraints violated", len(warn_msgs))
+                log.info("%s Warning only constraints violated", len(warn_msgs))
```

Comment:
This probably suppresses too much but we can figure out a better strategy later (maybe only warn this once per frame)